README.md

```diff
@@ -37,7 +37,7 @@ C4 can accommodate multiple concurrent audits, and sponsors are free to start an
 Since the goal of each participant is to find any vulnerabilities rather than all of them, C4 code reviews are ideal for highly specialized security researchers interested in demonstrating their skill and creativity. The more rare or novel the identified bug, the higher the reward for finding it.
 
 *Opportunity*
-Because everyone who finds vulnerabilities is compensated from the pool, C4 presents a great way for beginners to get compensated for their time learning how to hunt for smart contract exploits, to learn from other researchers, and to build a name and reputation within the community as their skills grow.
+Because everyone who finds vulnerabilities is compensated from the pool, C4 enables beginners to get paid to learn and to build a reputation within the community as their skills grow.
```

Comment:
We don't want to limit this exclusively to beginners though, so I think we should add a blurb for more experienced auditors/researchers (either in a separate section or appended to this sentence.)

Reply:
@zscole good point. let me do that.